### PR TITLE
Schema upgrade: 1.3.0

### PR DESCRIPTION
**Pull request by this-username-is-taken**
### Common

- Added revenue data to pools.
- Added more networks.
- Fixed markdown formatting issue in comments (# gets interpreted as heading)

### DEXes

- Added `isSingleSided` for single-sided pools. Single-sided pools should be able to reuse/adopt most if not all of the existing fields with minor adaptation. These should be handled on a per protocol basis.

### Lending

- Changed `fixed_term` to `fixed` in `InterestRateType` entity to be more precise.

### CODE REVIEW

**Comment by steegecs** [comment]
The updates on the DEX's at least look good to me.

**Comment by nemani** [comment]
Works for us!

File: schema-dex-amm.graphql [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 enum Network {
   ARBITRUM_ONE
+  ARWEAVE_MAINNET
   AURORA
   AVALANCHE
+  BOBA
   BSC # aka BNB Chain
   CELO
+  COSMOS
+  CRONOS
```
Comment (dmelotik) [inline schema-dex-amm.graphql:14]: Should `Clover` be added here too? https://parachains.info/details/clover_finance

Comment (this-username-is-taken) [reply schema-dex-amm.graphql:14]: hmm, I can't even find Clover in Defillama. Are they relevant?

File: schema-dex-amm.graphql [hunk 2]
```diff
@@ -92,6 +99,12 @@ enum LiquidityPoolFeeType {
 
   " Some protocols use dynamic protocol fees (e.g., Bancor v2). Set `feePercentage` as 0 but handle the dynamic fees in the mapping code. "
   DYNAMIC_PROTOCOL_FEE
+
+  " One-time fee charged by the protocol during deposit, in percentages of the deposit token "
+  DEPOSIT_FEE
+
+  " One-time fee charged by the protocol (e.g. Bancor) during withdrawal, in percentages of the withdrawal token "
```
Comment (0xbe1) [inline schema-dex-amm.graphql:106]: Bancor -> Bancor V3 to be clear

**Comment by 0xbe1** [comment]
dex amm schema changes look good

**Pull request merged**